config/locales/en.yml

```diff
@@ -1409,7 +1409,7 @@ en:
         clar_page_intro_html: |
           If needed, please complete an
           <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Special Prep Forms">appropriate special prep form</a>
-          and/or <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">Unused materials (over 3 hours) form</a>
+          or <a href="https://www.gov.uk/government/publications/af1-claim-for-advocate-graduated-fees" target="_blank" rel="external" title="Unused materials (over 3 hours) form">unused materials (over 3 hours) form</a>
```

Comment:
@jsugarman ps - you don't need the title attribute as the link text is almost the same and already descriptive.

Reply:
this is just a minor content change to existing stuff, can you add a commit for the visually hidden stuff or put an example in here